app/components/provider_interface/check_course_details_component.rb

```diff
@@ -2,10 +2,10 @@ module ProviderInterface
   class CheckCourseDetailsComponent < ChangeCourseDetailsComponent
     def rows
       [
-        { key: 'Training provider', value: course_option.provider.name_and_code, action: { href: change_provider_path } },
+        { key: 'Training provider', value: course_option.provider.name, action: { href: change_provider_path } },
         { key: 'Course', value: course_option.course.name_and_code, action: { href: change_course_path } },
         { key: 'Full time or part time', value: course_option.study_mode.humanize, action: { href: change_study_mode_path } },
-        { key: 'Location', value: course_option.site.name_and_address, action: { href: change_location_path } },
+        { key: 'Location', value: course_option.site.name_and_address.gsub(', ', "\n"), action: { href: change_location_path } },
```

Comment:
instead of doing a `gsub` everywhere we call `name_and_address` we could add a new method to site which does this and use that instead?